readme.md

```diff
@@ -166,18 +166,6 @@ gem "pry"  # this gem allows access to REPL
 
 Then I'm going to run `$ bundle install` in the terminal.
 
-### Setup Ruby - Tunr (You Do - 10 / 70)
-If you did not complete the initial SQL setup for Tunr above:
- - In your cloned copy, run `git fetch https://github.com/ga-dc/tunr_sinatra` to get the most recent branches
- - Please `git checkout` to the `2_active_record_starter` branch
- - Then you will need to create the db with `createdb tunr_db`
- - Then load the schema file with `psql -d tunr_db < db/schema.sql`
- - Next run the seed file with `psql -d tunr_db < db/seed.sql`
- - Follow the steps from `Part 2.1` in the link below
-
-[Part 2.1 - Create the Artist Model Using Active Record](https://github.com/ga-dc/tunr_sinatra/tree/2_active_record_starter#part-21---create-the-artist-model-using-active-record)
-
```

Comment:
Much better to move this to the initial setup for [tunr](https://github.com/ga-wdi-exercises/tunr-active-record#part-11---create-the-artist-model-using-active-record). Do they already do the first part in SQL class, or do you need to intro it here?